### ISSUE TITLE
NoSuchFieldError in  2017.1 EAP

### ISSUE THREAD

**Issue by channingwalton** [concordion/idea-concordion-support#13]
IntelliJ IDEA (Community Edition) 2017.1 EAP
Build #IC-171.3566.24, built on February 21, 2017

```
Instance
com.intellij.openapi.extensions.impl.PicoPluginExtensionInitializationException: Instance
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:96)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:119)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:246)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:191)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:102)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:89)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:50)
	at org.concordion.plugin.idea.ConcordionExtension.extensions(ConcordionExtension.java:63)
	at org.concordion.plugin.idea.fixtures.ConcordionTestFixtures.fixtures(ConcordionTestFixtures.java:45)
	at org.concordion.plugin.idea.menu.ConcordionSpecMenu.<init>(ConcordionSpecMenu.java:30)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.intellij.util.ReflectionUtil.newInstance(ReflectionUtil.java:419)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.convertStub(ActionManagerImpl.java:152)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getActionImpl(ActionManagerImpl.java:485)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getAction(ActionManagerImpl.java:473)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.preloadActions(ActionManagerImpl.java:1248)
	at com.intellij.openapi.actionSystem.impl.ActionPreloader.preload(ActionPreloader.java:34)
	at com.intellij.openapi.application.Preloader.lambda$null$0(Preloader.java:74)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:163)
	at com.intellij.openapi.application.Preloader.lambda$initComponent$1(Preloader.java:72)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoSuchFieldError: Instance
	at org.concordion.plugin.idea.fixtures.ConcordionScalaTestFixture.<init>(ConcordionScalaTestFixture.java:17)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:103)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:80)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:63)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:77)
	... 30 more
Caused by:

java.lang.NoSuchFieldError: Instance
	at org.concordion.plugin.idea.fixtures.ConcordionScalaTestFixture.<init>(ConcordionScalaTestFixture.java:17)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:103)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:80)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:63)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:77)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:119)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:246)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:191)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:102)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:89)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:50)
	at org.concordion.plugin.idea.ConcordionExtension.extensions(ConcordionExtension.java:63)
	at org.concordion.plugin.idea.fixtures.ConcordionTestFixtures.fixtures(ConcordionTestFixtures.java:45)
	at org.concordion.plugin.idea.menu.ConcordionSpecMenu.<init>(ConcordionSpecMenu.java:30)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.intellij.util.ReflectionUtil.newInstance(ReflectionUtil.java:419)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.convertStub(ActionManagerImpl.java:152)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getActionImpl(ActionManagerImpl.java:485)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getAction(ActionManagerImpl.java:473)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.preloadActions(ActionManagerImpl.java:1248)
	at com.intellij.openapi.actionSystem.impl.ActionPreloader.preload(ActionPreloader.java:34)
	at com.intellij.openapi.application.Preloader.lambda$null$0(Preloader.java:74)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:163)
	at com.intellij.openapi.application.Preloader.lambda$initComponent$1(Preloader.java:72)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by gmandnepr** [comment]
Taking a look.

Do you have markdown/scala/groovy plugin installed?

**Comment by channingwalton** [comment]
I have the scala plugin installed.

**Comment by gmandnepr** [comment]
Is it 2017.1.8?

**Comment by gmandnepr** [comment]
[IdeaConcordionSupport-0.12-SNAPSHOT.zip](https://github.com/concordion/idea-concordion-support/files/812100/IdeaConcordionSupport-0.12-SNAPSHOT.zip)

Could you please confirm if this build works well?

**Comment by channingwalton** [comment]
Your snapshot works!

fyi I'm using markdown navigator 2.3.4.2 and scala 2017.1.8.6

**Comment by gmandnepr** [comment]
:+1: 

I will upload new release on jetbrains plugin repository by the weekend.

**Comment by channingwalton** [comment]
Thanks for the quick response. And a very useful plugin.

**Comment by gmandnepr** [comment]
Fixed in version 0.11.1 (idea should notify about update soon)

In case you have SNAPSHOT build installed from the link above, you may need to uninstall it and install from plugin repository manually.

**Comment by channingwalton** [comment]
Great, thanks.
